controllers/credentials.js

```diff
@@ -44,6 +45,22 @@ passport.use(new GoogleTokenStrategy({
 
 */
 
+exports.createUser = async function (req, res) {
```

Comment:
Can we put this in a new controller `controllers\users.js`

Reply:
done